banner/templatetags/banner_tags.py

```diff
@@ -1,43 +1,46 @@
+import six
+
 from django import template
 
+from banner.models import Banner
+from banner.styles import BANNER_STYLES_MAP
+
 register = template.Library()
 
 
 @register.tag
-def get_actual_banner(parser, token):
-    """Get actual banner for a BannerProxy object.
-
-    Syntax::
-
-        {% get_actual_banner for [object] as [varname] %}
-    """
-    tokens = token.contents.split()
-    if len(tokens) != 5:
-        raise template.TemplateSyntaxError(
-            "%r tag requires 4 arguments" % tokens[0]
-        )
+def render_banner(parser, token):
+    tokens = token.split_contents()
 
-    if tokens[1] != 'for':
+    if len(tokens) < 2:
         raise template.TemplateSyntaxError(
-            "First argument in %r tag must be 'for'" % tokens[0]
+            "render_banner tag requires at the slug or the banner object."
```

Comment:
This message seems a bit off.

Reply:
Rather make the error something like:
```
        raise template.TemplateSyntaxError(
            Tag usage: "{% render_banner <slug>/<object> %}"
        )
```
Re-indicate correct usage somehow.

Reply:
Whoops, looks like a left out a few words in that message :see_no_evil: